apps/framework-cli/src/framework/data_model/schema.rs

```diff
@@ -130,6 +138,12 @@ impl Serialize for ColumnType {
                 state.serialize_field("elementType", inner)?;
                 state.end()
             }
+            ColumnType::Nested(nested) => {
+                let mut state = serializer.serialize_struct("Nested", 2)?;
```

Comment:
need to change `fn visit_map` as well
maybe we should add a tag to the JSON object.

Reply:
What does tag mean in this case? A parent? A unique ID?